src/components/Newsletter/Newsletter.js

```diff
@@ -202,13 +201,17 @@ class Newsletter extends React.Component {
               maxlength="200"
               name="cm-yddtsd-yddtsd"
               required=""
+              pattern="^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$"
```

Comment:
Hrmmm...  Why is this a different regex?  And is it actually used?

Reply:
ah ya i was hopping this would just...work but it didn't seem to do anything